io.github.Ethanscharlie.albumripper.json

```diff
@@ -0,0 +1,40 @@
+{
+    "id" : "io.github.Ethanscharlie.albumripper",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "47",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "albumripper",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+	"python3-requests.json",
+        {
+            "name" : "albumripper",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/Ethanscharlie/albumripper.git",
+        	    "tag": "v1.0.1"
+                }
```

Comment:
indentation is broken

<img width="726" height="356" alt="Image" src="https://github.com/user-attachments/assets/1f8755f9-ad53-4e5f-bf12-d45895c8203e" />